sql/parse/parse_test.go

```diff
@@ -788,6 +789,34 @@ var fixtures = map[string]sql.Node{
 		},
 		plan.NewUnresolvedTable("bar", "foo"),
 	),
+	`UNLOCK TABLES`: plan.NewUnlockTables(),
```

Comment:
could you test table names with numbers?

Reply:
Done